src/Symfony/Bridge/Twig/Tests/Extension/HttpKernelExtensionTest.php

```diff
@@ -46,7 +46,13 @@ public function testUnknownFragmentRenderer()
         ;
         $renderer = new FragmentHandler(array(), false, $context);
 
-        $this->setExpectedException('InvalidArgumentException', 'The "inline" renderer does not exist.');
+        if (method_exists($this, 'expectException')) {
+            $this->expectException('InvalidArgumentException');
```

Comment:
For reference: https://thephp.cc/news/2016/02/questioning-phpunit-best-practices
Let's keep it this way then.